hooks/frame_operations_tk-3dsmax.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2013 Shotgun Software Inc.
```

Comment:
I appreciate you've setup the hooks like for like with the original behaviour, but we don't support the tk-3dmax engine anymore, so this file can be removed.